packages/models-library/src/models_library/api_schemas_catalog.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+from .services import ServiceKey, ServiceVersion
+
+
+class ServiceAccessRightsGet(BaseModel):
+    service_key: ServiceKey
+    service_version: ServiceVersion
+    gids_with_access_rights: dict[int, dict[str, bool]]
```

Comment:
replace with ```GroupID, ServiceAccessRights``` or something similar?

Reply:
replaces with GroupID, but regarding the value having it as the dictionary is pretty useful during the checks. Maybe I can add some constrained string or leave it like that for now.

Reply:
maybe check that one https://github.com/ITISFoundation/osparc-simcore/blob/dfdd1c4669e985ea68e8e8489b42defae40c13b3/packages/models-library/src/models_library/services_access.py

Reply:
Aa thanks sorry I missed that, I will create a small separate PR for that I would like to merge this guy now so Odei can start to implement.